bimsb/packages/bioinformatics-nonfree.scm

```diff
@@ -1882,3 +1883,72 @@ MACE is a bioinformatics tool dedicated to analyze ChIP-exo data.")
     ;; others say "free for all use", others say "GPL version 1.3" (?), yet
     ;; others say "Artistic license".
     (license license:gpl2+)))
+
+;; Although this program is released under the GPL it depends on
+;; ViennaRNA and bowtie1, which are non-free software.
+(define-public mirdeep2
+  (package
+    (name "mirdeep2")
+    (version "0.0.92")
```

Comment:
This will need to be bumped before we squash, rebase & merge.

Reply:
done in https://github.com/BIMSBbioinfo/guix-bimsb-nonfree/pull/1/commits/103168f2fe4de4096d22241effc70ef97b3be655